Source/Cvent.SchemaToPoco.Core/JsonSchemaResolver.cs

```diff
@@ -287,6 +302,25 @@ private string StandardizeReferences(Uri parentUri, string data, ICollection<str
                 {
                     var matched = pattern.Match(lines[i]);
                     var matchedPath = matched.Groups[2].Value;
+                    if (matchedPath.StartsWith("#"))
+                    {
+                        //JSON pointer syntax isn't handled properly, but by creating our .json files with a convention
+                        //that the last part of the pointer path matches the ID of the schema being declared in the same file,
+                        //the reference can be fixed up by changing it to that ID (while still allowing the JSON pointer to work for Java)
+                        string[] parts = matchedPath.Split('/');

```

Comment:
nope. because it has to deal with stuff like this: `#/properties/activityDetails`